src/sagemaker_huggingface_inference_toolkit/mms_model_server.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2021 The HuggingFace Team, Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I might be missing something here, why do we have this file again inside `src` ?

Reply:
yes was a mistake. only the once in `src` should exist removed all others